File: curso-python/ex060.py
# faça um programa que leia um número qualquer e mostre seu fatorial

# ex:
# 5! = 5x4x3x2x1 = 120

n = int(input("digite um número para calcular seu fatorial:"))
contador = n
fat = 1

print(f"{n}! = ", end="")

while contador > 0:
    print(f"{contador}", end="")
    print(" x " if contador > 1 else " = ", end="")
    fat *= contador
    contador -= 1

print(fat)

File: javascript-exercicios/ex060.js
const prompt = require('prompt-sync')();

let n = parseInt(prompt("Digite um número para calcular seu fatorial: "));
let contador = n;
let fat = 1;

process.stdout.write(`${n}! = `);

while (contador > 0) {
    process.stdout.write(`${contador}`);
    if (contador > 1) {
        process.stdout.write(" x ");
    } else {
        process.stdout.write(" = ");
    }
    fat *= contador;
    contador--;
}

console.log(fat);
